src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/DocFinder.java

```diff
@@ -265,11 +265,15 @@ public static Output search(BaseConfiguration configuration, Input input) {
                 inheritedSearchInput.element = superclass;
                 output = search(configuration, inheritedSearchInput);
                 output.isValidInheritDocTag = true;
-                if (!output.inlineTags.isEmpty()) {
+                if (isNonEmpty(output.inlineTags)) {
                     return output;
                 }
             }
         }
         return output;
     }
+
+    private static boolean isNonEmpty(List<? extends DocTree> list) {
+        return list != null && !list.isEmpty();
```

Comment:
`output.inlineTags` should never be null. Separately, can this readability change be done in a separate PR? I do have one suitable PR in the works, which can include it.

Reply:
I'll back out these lines as long as the tests still pass.

Reply:
done